README.rst

```diff
@@ -89,3 +97,12 @@ Linux
 Run::
 
     python mte/gui.py
+
+Running environment from virtual machine as Host system
+----------------------------------------
+It is possible to run the testing environment from host system being virtual machine. In this case it is
+required to comment out the following line in ``virtual.py`` module::
+
+    start_machine()
+
+The guest virtual machine has to be booted up already when the testing is run through GUI.
```

Comment:
This last note is not clear to me. Do you imply that if you want to test through GUI, you have to start up the VM manually?

Reply:
Yes, exactly.

Reply:
But there's functionality to boot the VM automatically. Is there anything wrong with it?

Reply:
Honestly I haven't tried, so I cannot tell. I guess it is not clear so I will think about better description later. 
> It is possible to run the testing environment from host system **being virtual machine**. 

The section is about running the MTE where both guest and host are being virtual machines. Imagine it like this:
```
Windows OS
|
| - - - - - - - - - - - - - - VM 1 (Host scripts)
|                                        ^
|                                        |
|                                        |
|                                        v
| - - - - - - - -  - - - - - -VM 2 (Guest scripts)
```
Yes of course you can boot the VM2 from the Windows OS using vbox_api and function start_machine() but that section describes something else.

In this scenario, you cannot control the VM2 with VM1 through vbox_api because VM2 is not running inside VM1. Both VM1 and VM2 are running inside Virtualbox in Windows OS. Because it does not make much sense to try to run Virtualbox inside Virtual machine booted inside Virtualbox (such as in my case) I wrote alternative solution that I used, for people who may want to use the same approach.

I guess the confusing part is that I named the virtual machine being 'Host' while it is not exactly true, but I did it because everywhere in the README, we refer to a machine, which is making SSH connection as Host machine.

Reply:
Oh right, my mistake. I thought we're talking about just Windows host and Linux VM.

Maybe I forgot, but why exactly do you have to use special VM 1 for host scripts when they can run on Windows host machine?

Reply:
I don't have to.
> **It is possible to run** the testing environment from host system being virtual machine. **In this case** it is required to comment out the following line in ``virtual.py`` module::

I just like to keep my Host system clean without polluting it with a lot of different devtools, so I prefer using VM instead.